configs/sites/tier1/nas/packages.yaml

```diff
@@ -0,0 +1,94 @@
+packages:
+  all:
+    target: [x86_64_v3]
+### Modification of common packages
+  # Problems building shared hdf-eos2 with Intel, not needed
+  hdf-eos2:
+    variants: ~shared
+  met:
+    variants: +python +grib2 +graphics +lidar2nc +modis
+### All other external packages listed alphabetically
+  wget:
+    externals:
+    - spec: wget@1.19.5
+      prefix: /usr
+  perl:
+    externals:
+    - spec: perl@5.26.3~cpanm+opcode+open+shared+threads
+      prefix: /usr
+  sed:
+    externals:
+    - spec: sed@4.5
+      prefix: /usr
+  grep:
+    externals:
+    - spec: grep@3.1
+      prefix: /usr
+  bison:
```

Comment:
Surprised you didn't run into duplicates for this. I had to remove any external bison older than 3.4 from the site configs that I manage, because some package insists on a version 3.4+ or newer and then I ended up with the external 3.0 and a spack-built 3.8.

Reply:
Huh. Yeah, I haven't so far I guess.